### PR TITLE
DataGrid: align ColumnChooser with Pager elements

### LINKED ISSUE #5587: [Bug]: ColumnChooser is not aligned with pager
### Blazorise Version

1.5.3

### What Blazorise provider are you running on?

FluentUI2

### Link to minimal reproduction or a simple code snippet

Example from doc's

### Steps to reproduce

<DataGrid TItem="Employee"
          Data="inMemoryData"
          Responsive
          ShowColumnChooser
          ShowPager
          ShowPageSizes>
</DataGrid>

### What is expected?

ColumnChooser is not aligned with pager

### What is actually happening?
image

example from demo:
image

### What browsers do you see the problem on?

_No response_

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by steal4life**
closes #5587

### CODE REVIEW

File: Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿@typeparam TItem
 @inherits BaseComponent
 <Row class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
-    <Column ColumnSize="ColumnSize.IsAuto" Flex="Blazorise.Flex.Row.AlignItems.Center">
+    <Column ColumnSize="ColumnSize.IsAuto" Flex="Blazorise.Flex.Row">
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:4]: I believe that was a reason why `.AlignItems.Center` was used? @David-Moreira do you remember why? I think have added it.

Comment (David-Moreira) [reply Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:4]: Ah it was sometime ago, but I remember you refactored it, so it was consistent across providers?

We went back and forth whether we should make it all use flex. And you used AlignItems.Center on your refactor, however there's no mention of the why, but we've tested and it was working fine.

**PR:**
https://github.com/Megabit/Blazorise/pull/5324
**Commit that introduced it:**
https://github.com/Megabit/Blazorise/pull/5324/commits/567ea0031c6b1e26fcf3503f7156fb71a699aeb4

Comment (stsrki) [reply Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:4]: Ah yes, I remember. Ok I will see if it makes sense to leave it.

File: Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor [hunk 2]
```diff
@@ -117,7 +117,7 @@
     {
         @if ( ShowColumnChooser )
         {
-            <Field Display="Blazorise.Display.InlineBlock" Margin="Blazorise.Margin.IsAuto.OnY.Is2.FromStart">
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:120]: I think we should only remove Y margin, and leave the rest `Margin="Blazorise.Margin.Is2.FromStart"`

Comment (tesar-tech) [reply Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:120]: With from start it looks like:
image

So I keep it on both sides. 
image

Comment (stsrki) [reply Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:120]: It seems to me that the BS4 is now broken.

image

vs live demo (https://bootstrapdemo.blazorise.com/tests/datagrid/column-chooser)

image

---

Same for BS5. Haven't look at other providers.

Comment (tesar-tech) [reply Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor:120]: The column chooser button needs different margins on different positions, thus I made it so.

**Review by stsrki — APPROVED** [review]

**Pull request merged**